Lesson9/main.js

```diff
@@ -0,0 +1,47 @@
+const ul = document.getElementById("js-ul");
+const loading = document.getElementById("js-loading")
+const features = [{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+{to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}];
+
+const addLoading = () =>{
+  const loadingImg = document.createElement('img');
+  loadingImg.src = 'loading.gif';
+  loadingImg.id = "loading-img";
+  loading.appendChild(loadingImg);
+};
+
+const removeLoading = () =>{
+  const removeImg = document.getElementById('loading-img')
+  removeImg.remove();
+};
```

Comment:
IMO
I didn't think it needed to be a variable if it wasn't going to be used repeatedly.
```suggestion
const removeLoading = () => document.getElementById('loading-img').remove();
```

Reply:
thank you for the comment. I refactored the line